account_cost_center/models/account_invoice_report.py

```diff
@@ -7,21 +7,10 @@
 class AccountInvoiceReport(models.Model):
     _inherit = "account.invoice.report"
 
-    cost_center_id = fields.Many2one(
-        "account.cost.center", string="Cost Center", readonly=True
-    )
+    cost_center_id = fields.Many2one("account.cost.center", readonly=True)
 
     def _select(self):
-        return (
-            super(AccountInvoiceReport, self)._select()
-            + ", sub.cost_center_id as cost_center_id"
-        )
-
-    def _sub_select(self):
-        return (
-            super(AccountInvoiceReport, self)._sub_select()
-            + ", ail.cost_center_id as cost_center_id"
-        )
+        return super()._select() + ", line.cost_center_id as cost_center_id"
```

Comment:
This line is missed by coverage. Could we add a simple test to just `search_read` from `account.invoice.report`? I believe that should be enough to exercise this line.